packages/xstate-react/test/types.test.tsx

```diff
@@ -171,10 +171,14 @@ describe('useMachine', () => {
 
     function Element({ myActor }: Props) {
       const [current, send] = useActor(myActor);
+      const bar: number = current.context.bar;
```

Comment:
Why is the `: number` assertion needed?

Reply:
Oh, its not really needed - i have wanted to check assignability of this (so kinda inference by extension) but it could always to be any so this check is not that great :/